### ISSUE TITLE
small code coverage decreases (-0.0008%) put a red X on pull requests

### ISSUE THREAD

**Issue by pdurbin** [IQSS/dataverse#7321]
Pull requests can either show a green checkmark (GOOD!) or a red X (BAD!). Here's an example of the latter:

<img width="347" alt="Screen Shot 2020-10-09 at 1 54 09 PM" src="https://user-images.githubusercontent.com/21006/95615908-f7fd0c80-0a36-11eb-80ed-4c1cfe9b73a9.png">

If you dig into this pull request, you'll find that automated API tests ran find but that code coverage was reduced by -0.0008%, which is enough to deliver the red X:

<img width="771" alt="Screen Shot 2020-10-09 at 1 51 31 PM" src="https://user-images.githubusercontent.com/21006/95615913-f92e3980-0a36-11eb-8431-2e75b3585570.png">

This problem isn't new. It's been around since I set up Coveralls in d6444d3. What's new is that I finally got around to finding what I hope is a fix, which is described at https://github.com/deepchem/deepchem/issues/648

In short, I think we need to...

- decide on a reasonable threshold to allow
- enter that number into the the Coveralls "Coverage Decrease Threshold for failure" (screenshot below)

<img width="821" alt="Screen Shot 2020-10-09 at 2 05 27 PM" src="https://user-images.githubusercontent.com/21006/95616877-8d4cd080-0a38-11eb-8c09-6b32b9ae1149.png">

**Comment by djbrooke** [comment]
- Unit Tests 
- .5 percent as possibility 
- Metrics PR that was a bunch of code changes only dropped it .2 percent

**Comment by pdurbin** [comment]
I went to https://coveralls.io/github/IQSS/dataverse/settings and set "Coverage Decrease Threshold for failure" to 0.5%. (Previously it was unset.) Now I guess we wait until someone makes a pull request.

<img width="1356" alt="Screen Shot 2020-11-06 at 10 18 34 AM" src="https://user-images.githubusercontent.com/21006/98382838-b6518880-2019-11eb-95a3-6ae3de467eac.png">
<img width="1356" alt="Screen Shot 2020-11-06 at 10 18 59 AM" src="https://user-images.githubusercontent.com/21006/98382844-b8b3e280-2019-11eb-8f3c-0c09bbac4a8b.png">

**Comment by pdurbin** [comment]
I think we're good. Here's an example of a green check next to Coveralls:

<img width="923" alt="Screen Shot 2020-11-17 at 10 01 12 AM" src="https://user-images.githubusercontent.com/21006/99440613-d2400e80-28e4-11eb-8718-97606102cc15.png">

In pull request #7421 I made a note of the change in the dev guide.
